app/src/models/pull-request.ts

```diff
@@ -20,6 +25,8 @@ export class PullRequestRef {
 }
 
 export class PullRequest {
+  public parsedMarkdownBody?: MarkdownEmitter
```

Comment:
Not a big fan of having this in the "raw" Pull Request model, to be honest 🤔 Once it's initialized, it'll stay in memory until it's removed from the PullRequestStore, right?

Maybe it makes more sense to put it in the PullRequestListItem? 🤔